tests/slack-import-test-posts.json

```diff
@@ -176,5 +176,41 @@
         "username": "<@U07Q4MHCP|lindy>",
         "bot_id": null,
         "ts": "1469782077.000010"
+    },
+    {
+        "type": "message",
+        "text": "and an archive",
+        "files": [
+            {
```

Comment:
nit: Could we test `files` with more than one entry in the array?

Reply:
hey @lieut-data I replaced this with a new post from a fresh export that has more than one attachment.

I also added an additional test for the attachments, to test if the correct number of attachments was parsed. lmk if you have any suggestions for improvements on this.